cli/options/options.go

```diff
@@ -194,6 +196,10 @@ func GetRPCWithInvoker(gctx context.Context, ctx *cli.Context, signers []transac
 // GetConfigFromContext looks at the path and the mode flags in the given config and
 // returns an appropriate config.
 func GetConfigFromContext(ctx *cli.Context) (config.Config, error) {
+	if ctx.NumFlags() == 0 {
+		return getDefaultVMConfig()
+	}
+
```

Comment:
Please, do not mix VM config with this, you don't need this change. VM CLI is perfectly OK with the existing code and embedded node config, it doesn't require any changes with the updated design.